app/src/main/java/org/mozilla/focus/browser/HomeTileRecyclerView.kt

```diff
@@ -44,7 +44,8 @@ private val CUSTOM_TILE_ICON_INTERPOLATOR = DecelerateInterpolator()
 class HomeTileAdapter(
```

Comment:
nit: File name does not represent what's in the file: can you fix the name (maybe in the last commit so it's easier to read the history)?